linode/manager
# Hide broken stories and fix paginated table example

## Description

Very rough review of the Storybook entries.
Changes include:
- Hide `EntityDetail` and `NotificationDrawer` stories since they're breaking
- Prevent stories from nesting if it's the only story for the component
- Use `LinodeRow` for the paginated table story instead to prevent the content from wrapping

## How to test

- Run `yarn storybook`
- Check the relevant stories
